### DISCUSSION TITLE
[Suggestion] Add more husk versions of existing creatures

### DISCUSSION THREAD

**Discussion by StruggleDuck** [FakeFishGames/Barotrauma#6847] [Ideas and suggestions]
Currently, the only creatures capable of becoming husked are humans and crawlers, and I feel that it would make much more sense if there was husk variants of things like mudraptors and threshers. I feel like this would really make husks more threatening, and would add more enemy variety.

**Comment by SpazzMaticus107** [comment]
I'd love to see the devs 1up-ing huskmod. The shambler head that can detach after the body is dead, jump at the human and take over it's body amazes me every time.
